docs/reference/bigconfig.md

```diff
@@ -24,13 +24,13 @@ This includes actions such as creation, update, and deletion of Bigeye metrics.
 ## Anatomy of a bigConfig file
```

Comment:
Is there a specific reason for the below changes? The format used is valid and works?